core/src/utilities/helpers.js

```diff
@@ -86,6 +86,36 @@ export const prependOrigin = path => {
   return window.location.origin;
 };
 
+export const containsAllSegments = (sourceUrl, targetPathSegments) => {
```

Comment:
There is still a bug when the user introduces a url with a trailing slash like '/projects/pr2/' (`sourceUrl` param in function)
Furthermore the implementation could be simpler. My proposal is [the following](https://github.com/kyma-project/luigi/files/2594680/404-support-for-non-existing-paths-2.zip) (unit tests are still green after refactoring)